public/components/data_connections/components/acceleration_ui/management/management_table.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiSpacer,
+  EuiTableFieldDataColumnType,
+  EuiText,
+} from '@elastic/eui';
+import _ from 'lodash';
+
+interface AccelrationIndicesRecordType {
```

Comment:
typo? same for `accelrationIndicesRecords, setAccelrationIndicesRecords`
```suggestion
interface AccelerationIndicesRecordType {
```

Reply:
Thanks for the catch!